CONTRIBUTING.md

```diff
@@ -163,22 +201,42 @@ $ git lfs pull
 
 [git-lfs]: https://git-lfs.github.com
 
-## What to work on
+## Issues
+
+The [GitHub tracker][issu] is our canonical source of truth around issues,
+bugs, performance problems, feature requests, and so on.  If you encounter any
+issues when developing on urbit, feel free to submit a report about it here.
 
-If you are not thinking of contributing with a specific goal in mind, the
-GitHub issue tracker is the first place you should look for ideas.  Issues are
-occasionally tagged with a priority and a difficulty; a good place to start is
-on a low-difficulty or low-priority issue.  Higher-priority issues are likely
-to be assigned to someone -- if this is the case, then contacting that person
-to coordinate before starting to work is probably a good idea.
+You should label your issue appropriately if you know happen to know what it
+applies to.  For example, a `%gall` bug should be labelled with both 'gall' and
+'bug'.  It's ok if you don't know how to label your issue -- the maintainers
+will help you out and try to label it appropriately.
```

Comment:
Random contributors can't actually add/modify labels (or anything else) themselves, I don't think. We'll always have to do that for them.